deckhouse-controller/pkg/debug/module_enable.go

```diff
@@ -0,0 +1,81 @@
+package debug
+
+import (
+	"fmt"
+	"net/http"
+
+	addon_operator "github.com/flant/addon-operator/pkg/addon-operator"
+	sh_app "github.com/flant/shell-operator/pkg/app"
+	sh_debug "github.com/flant/shell-operator/pkg/debug"
+	"github.com/go-chi/chi/v5"
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	deckhouse_config "github.com/deckhouse/deckhouse/go_lib/deckhouse-config"
+)
+
+func RegisterModuleEnableRoutes(dbgSrv *sh_debug.Server, op *addon_operator.AddonOperator) {
+	dbgSrv.RoutePOST("/module/{name}/{action:(disable|enable)}", func(r *http.Request) (interface{}, error) {
+		modName := chi.URLParam(r, "name")
+		if modName == "" {
+			return nil, fmt.Errorf("'name' parameter is required")
+		}
+
+		action := chi.URLParam(r, "action")
+		switch action {
+		case "enable":
+			err := deckhouse_config.SetModuleConfigEnabledFlag(op.KubeClient, modName, true)
+			if err != nil {
+				return nil, fmt.Errorf("Enable module failed: %v", err)
+			}
+			return "Module enabled", nil
+		case "disable":
+			err := deckhouse_config.SetModuleConfigEnabledFlag(op.KubeClient, modName, true)
+			if err != nil {
+				return nil, fmt.Errorf("Disable module failed: %v", err)
+			}
+			return "Module disabled", nil
+		}
+		return nil, fmt.Errorf("Unknown action '%s' for module '%s'", action, modName)
```

Comment:
1. "For module" sounds like modules support different sets of actions which is not the case. 
2. I'd start with lower case
3. If path-wise action is not supported in HTTP, the handler should return 404

```go
return nil, fmt.Errorf("switching module %s: action '%s' is not supported", modName, action)
```

Reply:
There is no 'unknown action' error after update.